presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -548,6 +548,18 @@ private void addStatsListeners(QueryExecution queryExecution)
         });
     }
 
+    @Managed
+    public long getRunningTaskCount()
+    {
+        return queryTracker.getRunningTaskCount();
```

Comment:
Here it's referred to as a task count, so it's not consistent in this change set.